modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,247 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-understanding-terraform.adoc
+ifeval::["{context}" == "rosa-understanding-terraform"]
+:tf-full:
+endif::[]
+:_content-type: PROCEDURE
+
+[id="sd-terraform-account-roles_{context}"]
+ifdef::tf-full[]
+= Account roles Terraform example
+endif::tf-full[]
+ifndef::tf-full[]
+= Creating your account-wide IAM roles with Terraform
+endif::tf-full[]
+
+The following example shows how Terraform can be used to create your Amazon Web Services (AWS) Identity and Access Management (IAM) account roles for ROSA.
+
+[NOTE]
+====
+If you want to edit the Terraform files, you can use any text editor. You must re-run the `terraform init` and `terraform apply` commands if you change any values in the files.
+====
+
+.Procedure
+
+. Check your AWS account for existing roles and policies by running the following command:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
++
+
+
+. In your terminal, run the following command to export link:https://console.redhat.com/openshift/token[your OCM token]. This value must include the full OCM token:
```

Comment:
Likely too nitpicky, but I know it's been a thing before...suggestion: 

s/OCM/{cluster-manager-first}